src/transition.S

```diff
@@ -0,0 +1,62 @@
+.text
+.balign 4
+
+/* Wrapper to an intercept function, executed within a manager context;
+ * required in order to maintain a consisten execution state. This also manages
+ * setting the `DDC` as needed.
+ */
+.global intercept_wrapper
+.type intercept_wrapper, "function"
+// TODO restore c29
```

Comment:
That is quite important, yes :-)

Reply:
I think this comment doesn't apply anymore. This transition function is meant to be called via an `ldpblr` (particularly, the one in `compartment_transition_out`). As this function already saves `c29`, must we save it in this function as well?

Reply:
Yes, actually that's probably Ok. This function isn't PCS-compliant, but that's Ok as long as we don't call it from C :-)